torch/testing/_internal/common_methods_invocations.py

```diff
@@ -2272,6 +2288,49 @@ def _make_tensor_helper(shape, low=None, high=None):
                        SkipInfo('TestCommon', 'test_variant_consistency_jit',
                                 device_type='cuda', dtypes=[torch.float16]),
                    )),
+    UnaryUfuncInfo('sign',
+                   ref=np.sign,
+                   dtypes=all_types_and(torch.bfloat16, torch.half),
+                   dtypesIfCPU=all_types_and(torch.bool, torch.bfloat16, torch.half),
+                   dtypesIfCUDA=all_types_and(torch.bool, torch.bfloat16, torch.half),
+                   skips=(
+                       # Reference: https://github.com/pytorch/pytorch/issues/41245
+                       SkipInfo('TestUnaryUfuncs', 'test_reference_numerics_extremal',
+                                dtypes=[torch.bfloat16, torch.float16, torch.float32, torch.float64]),
+                       # >>> np.sign(True)
+                       # ufunc 'sign' did not contain a loop
+                       # with signature matching types dtype('bool') -> dtype('bool')
+                       SkipInfo('TestUnaryUfuncs', 'test_reference_numerics_normal',
```

Comment:
How do the other reference_numerics tests work for bool, then?

Should we wrap the reference so it works on the same inputs torch.sign does?

Reply:
Other functions work out-of-box with `bool`. Wrapping sounds ok for testing.
```python
>>> import numpy as np
>>> a = np.array(True)
>>> a.dtype
dtype('bool')
>>> np.sin(a)
0.8413
>>> np.abs(a)
True
```